EIPS/eip-7581.md

```diff
@@ -0,0 +1,68 @@
+---
+eip: 7581
+title: Increase block gas target and gaslimit
```

Comment:
it would be better so specify final target numbers here so as to be unambiguous w.r.t. future limit raising EIPs

Reply:
Done